### DISCUSSION TITLE
No css loading when accessing secret URL from Chrome on a mobile device

### DISCUSSION THREAD

**Discussion by vibarco** [pglombardo/PasswordPusher#1440] [General]
Hello,

Css is not loading when accesing secret link from a mobile device. It's working perfectly from workstation or desktop computer.

Best

**Comment by pglombardo** [comment]
Hi @vibarco - some questions:

1. Where?  pwpush.com or a self-hosted version?
2. If self-hosted, what version and is there a public URL I could test?
3. If self-hosted, what special configuration have you added?  Custom CSS?
4. What type of mobile device?  Andoid or IOS?
5. What browser is in use?
6. From the mobile device, is there a browser console that might give some indication or error?
7. Could you try a different browser on the mobile device?

Could you answer the above questions?  That should be the quickest way to get us closer to the core issue.

**Reply by pglombardo** [reply]
Hi @vibarco - did you make any progress on this issue?

**Comment by vibarco** [comment]
Hello Pete,

I have no time yet to test your recimmendantion. I'll try again as soon as
i can.

Best regards
